parcels/kernel.py

```diff
@@ -335,17 +337,17 @@ def execute(self, pset, endtime, dt):
                 stacklevel=2,
             )
 
-        if pset.fieldset is not None:
-            for g in pset.fieldset.gridset.grids:
-                if len(g._load_chunk) > g._chunk_not_loaded:  # not the case if a field in not called in the kernel
-                    g._load_chunk = np.where(
-                        g._load_chunk == g._chunk_loaded_touched, g._chunk_deprecated, g._load_chunk
-                    )
+        # if pset.fieldset is not None:
```

Comment:
This didn't seem to be needing to execute for the UXArray data, so I've just commented this out. Not sure what the plans are here, given that I recall us discussing removing chunking and deferred loading.

Reply:
This is gone in some of the recent updates